igvm/templates/aws_user_data.cfg

```diff
@@ -0,0 +1,37 @@
+#cloud-config
+
+cloud_config_modules:
+  - apt-configure
+  - runcmd
+
+manage_etc_hosts: localhost
+hostname: {{ hostname }}
+fqdn: {{ fqdn }}
+
+apt:
+  sources:
+    {{ innogames_stable_name }}:
+      source: "{{ innogames_stable_source }}"
+      key: |
+        {{ innogames_stable_key }}
+    {{ basestretch_stable_name }}:
+      source: "{{ basestretch_stable_source }}"
+      key: |
+        {{ basestretch_stable_key }}
```

Comment:
You could also generate the sources list here from a loop over AWS_CONFIG['apt'] to not even hardcode the two repos

Reply:
This is now generated with a loop

Reply:
Looking good! <3